examples/rabbitmq-transaction-producer/rabbitmq_transaction_producer.out

```diff
@@ -3,9 +3,6 @@
 # Navigate to the directory that contains the
 # 'rabbitmq_transaction_producer.bal' file, and run the 'ballerina run' command below.
 
-$ ballerina run --experimental rabbitmq_transaction_producer.bal
+$ bal run rabbitmq_transaction_producer.bal
```

Comment:
Missing `--experimental` flag?

Reply:
You don't need `--experimental` to run this.